src/Lua.cpp

```diff
@@ -97,7 +97,7 @@ static void get_host_vlan_info(char* lua_ip, char** host_ip,
   if(((*host_ip) = strtok_r(buf, "@", &where)) != NULL)
     vlan = strtok_r(NULL, "@", &where);
 
-  if(host_ip == NULL)
+  if(*host_ip == NULL)
```

Comment:
@simonemainardi 
Well, the previous behaviour implied checking if NULL and, if so, dereferencing it. This will be a NULL pointer dereference. 
So as i said:

> Either the condition 'host_ip==NULL' is redundant or there is possible null pointer dereference: host_ip